freetype.cmake

```diff
@@ -0,0 +1,18 @@
+set(FT_DISABLE_BROTLI ON CACHE STRING "Disable Brotli" FORCE)
```

Comment:
If you are adding a new cmake file it should go in `cmake/`. This is small enough that personally I would just put it in the main file.

Reply:
Why not. I did it that way because I mimicked how `rmlui.cmake` is stored in Unvanquished repository, but the Unvanquished repository has no `cmake/` folder (maybe it's time to do one there too).

Reply:
I will keep it that way for now, to be consistent with all other cmake files in both Unvanquished and Daemon repositories for building sources, like `src.cmake`, `srclibs.cmake` (and the `lua.cmake` and `rmlui.cmake` ones in Unvanquished).

We can define a new convention later and migrate those to a better name later, but for now this is consistent.